.gitignore

```diff
@@ -17,6 +17,7 @@ venv/
 
 /build
 /dash_core_components
+/dash_core_components_base/plotly.min.js
```

Comment:
Cleaned up generation a bit to copy file directly from node_modules instead of keeping a to-be-synced copy in the py generation base folder